src/Tasks/CreateManifestResourceName.cs

```diff
@@ -385,22 +379,28 @@ internal static string MakeValidEverettFolderIdentifier(string name)
         public static string MakeValidEverettIdentifier(string name)
         {
             ErrorUtilities.VerifyThrowArgumentNull(name, nameof(name));
+            if (!string.IsNullOrEmpty(name))
```

Comment:
with respect to my other nits, have our "base case" come first and return. Followed by all code that would run under normal circumstances

```suggestion
            if (string.IsNullOrEmpty(name))
            {
                   return;
            }
            
            <rest of code goes here with no else>
```